### DISCUSSION TITLE
Ubuntu 24.04 for arm64

### DISCUSSION THREAD

**Discussion by TimWolla** [ubicloud/ubicloud#1994] [Q&A] [answered]
We're using both x64 and arm64 runners in our CI pipeline. We'd like to update to Ubuntu 24.04 to benefit from the newer pre-installed tool versions (e.g. clang). Ubuntu 24.04 for x64 is available since quite a while (d997e64fadc15239ca065c3977071191a86e4d96), but for arm64 it is not, which unfortunately prevents us from upgrading.

Is there any ETA when Ubuntu 24.04 will become available for arm64?

**Comment by enescakir** [comment, marked as answer]
> We're using both x64 and arm64 runners in our CI pipeline. We'd like to update to Ubuntu 24.04 to benefit from the newer pre-installed tool versions (e.g. clang). Ubuntu 24.04 for x64 is available since quite a while ([d997e64](https://github.com/ubicloud/ubicloud/commit/d997e64fadc15239ca065c3977071191a86e4d96)), but for arm64 it is not, which unfortunately prevents us from upgrading.
> 
> Is there any ETA when Ubuntu 24.04 will become available for arm64?

Hi @TimWolla. If we don't encounter any major issues, we will enable it this week.

**Reply by TimWolla** [reply]
As far as I can tell based on https://github.com/ubicloud/ubicloud/blob/main/config/github_runner_labels.yml this did not happen last week.

Was there any unforeseen issues?

**Reply by enescakir** [reply]
We've been developing a new feature and decided to make it available for arm64 as well. Sorry for the delay. It's now deployed: https://github.com/ubicloud/ubicloud/pull/2039. 

I deployed it with limited capacity, but it should suffice for your needs.

**Reply by TimWolla** [reply]
Thank you. I can confirm it's working for my requirements :+1:
